07-reputation/04-globaltrust.md

```diff
@@ -0,0 +1,39 @@
+## Global Trust
+
+*Global trust*
+: the result of the [EigenTrust](http://ilpubs.stanford.edu:8090/562/1/2002-56.pdf) algorithm -- the trust in the network participant, which was obtained taking into account *all* local trusts of *all* nodes.
+
+### Subject and Object of a trust
+
+The Subject of the assessment is the entire system. Any node in the network is an Object of trust.
+
+The evaluation criterion is the aggregate of local trust of *all* network participants to the current node, but adjusted for similar local trust to each participant from the rest of the network.
+
+### Calculating trust
+
+Each manager retrieves the local trust value of its child at the end of each epoch and stores it in the `DaughterStorage`. The further task of each manager is to calculate the Global trust of the system in its child node, and also to transfer all the information he has to other managers so that they, in turn, can perform a similar task.
+
+Global trust is calculated iteratively. In the process of calculating `GlobalTrust`, according to the EigenTrust algorithm, managers need to exchange “intermediate” trust values, the so-called `IntermediateTrust`. With the help of these values, managers at each iteration approximate (recalculate the results of the previous iteration) the value of the Global trust to the common Global value limit: $T = [t_0, ..., t_n]$, where $t_i$ is the Global trust value in the node `i` limit.
```

Comment:
While calculating `GlobalTrust`, according to the EigenTrust algorithm, managers should exchange “intermediate” trust values... the limiting Global trust value in node `i`.

Reply:
fixed